serverless.yml

```diff
@@ -9,14 +9,15 @@ package:
 custom:
   esbuild:
     bundle: true
-    minify: true
+    minify: ${strToBool(${env:MINIFY, 'true'})}
```

Comment:
Checking that defaults to true if MINIFY is not set?

Reply:
It set minfy to `true` if MINIFY is not set.